### ISSUE TITLE
Speaker Bureau

### ISSUE THREAD

**Issue by claredillon** [InnerSourceCommons/InnerSourceMarketing#13]
Speaker Bureau is a list and a process to allow people to match qualified ISC / InnerSource speakers to speaking opportunities. 
Early version can be found here: http://innersourcecommons.org/contact/
Ask here is to re-look at and agree:
1. the process by which people get on the speaker bureau list (e.g. previous public experience talking about ISC?)
2. a starter list of people (e.g.everyone the old list still willing/still qualify?)
3. Process by which people request a speaker (e.g. do we list everyone publicly or get requests sent to a central place where we match best speaker with opportunity)
4. Assets people can use (e.g. shared "golden pitches" referred to by Danese). Note this can probably be completed without the rest of the complete project.

**Comment by lenucksi** [comment]
@dicortazar Please add your part on the assets

- Do we want to have a curated list of speaker or self-addition?
- How do we curate? @danese is in favor of curation 
- Intro-Assets to be re-used @danese says those are called "Golden Pitches"

- Speaker bureau channel
- Material to help people speaking out of their knowledge and to be able to deal with we controversial issues

Add more details here!

**Comment by spier** [comment]
@claredillon could you say more about what a Speaker Bureau is? I don't know the term.

**Comment by lenucksi** [comment]
> @claredillon could you say more about what a Speaker Bureau is? I don't know the term.

Also highlighting @dicortazar here since he came up with the first iteration. 
The wikipedia has this: https://en.wikipedia.org/wiki/Speakers_bureau which reasonably represents what I remember was being talked about .

**Comment by claredillon** [comment]
Speaker Bureau is a list of people in the ISC who are willing and able to speak on behalf of the ISC at third party events. An early list of people is published [here](http://innersourcecommons.org/contact/).


This card is related to an early discussion where it was agreed it needed more work, and we first specifically need to address the following issues/questions:

- Anyone can currently add themselves - but we need to decide if there is some qualifying criteria (e.g. you have to have presented at a summit). 
- We also need a process to match speakers to opportunities (curation)?
- Do we need to offer training?
- Do we need to offer standard presentations to be used? (Golden pitches referred above)? (Suggested yes - will include as separate card.

**Comment by gruetter** [comment]
> Anyone can currently add themselves - but we need to decide if there is some qualifying criteria (e.g. you have to have presented at a summit).

I think it depends on the process to match speakers. If all requests for a speaker will go through a process where s.o. matches them, they could also be the ones who do the qualification. Another option would be to team up inexperienced speakers with more experienced speakers, if that is logistically possible.

> We also need a process to match speakers to opportunities (curation)?

Danese mentioned that Apache had a process for this. Does anyone know this process?

> Do we need to offer training?

How about we offer potential speakers to help them review their presentation or participate in dress rehearsals? I think we have a couple of very experienced speakers in the commons.

> Do we need to offer standard presentations to be used? (Golden pitches referred above)? (Suggested yes - will include as separate card.

Depends very much on the speaker, I guess, but I'm sure standard presentations will be appreciated by enough speakers to make it worth the invest.

**Comment by lenucksi** [comment]
> > Anyone can currently add themselves - but we need to decide if there is some qualifying criteria (e.g. you have to have presented at a summit).
> 
> I think it depends on the process to match speakers. If all requests for a speaker will go through a process where s.o. matches them, they could also be the ones who do the qualification. Another option would be to team up inexperienced speakers with more experienced speakers, if that is logistically possible.

The idea to team up people sounds great!

> 
> > We also need a process to match speakers to opportunities (curation)?
> 
> Danese mentioned that Apache had a process for this. Does anyone know this process?

Can @MaineC or @danese add some insight here?

> 
> > Do we need to offer training?
> 
> How about we offer potential speakers to help them review their presentation or participate in dress rehearsals? I think we have a couple of very experienced speakers in the commons.

:+1: 

> 
> > Do we need to offer standard presentations to be used? (Golden pitches referred above)? (Suggested yes - will include as separate card.
> 
> Depends very much on the speaker, I guess, but I'm sure standard presentations will be appreciated by enough speakers to make it worth the invest.

:+1: 
We've had repeated request for such, there's even a ticket around this #27

**Comment by gruetter** [comment]
From Wikipedia:
> A speakers bureau helps client and speaker negotiate a speaking fee, a payment awarded to an individual for speaking at a public event. This fee is usually set by the speaker or the speaker’s agent. Logistics can be dealt with by the speakers bureau, like fees, transport, accommodation and timing, or communication between speaker and client.

One more task for the speakers bureau.

**Comment by claredillon** [comment]
Working on the Golden Pitch issue as a priority and will revert to formalizing this process once that is complete. https://github.com/InnerSourceCommons/marketing-wg/issues/27

Also - and linked to Georg's comment about speaker fees :) - the new sponsorship now includes the option of sourcing a speaker for internal events as a benefit - so will have to be addressed in the process for speakers.

**Comment by MaineC** [comment]
>> We also need a process to match speakers to opportunities (curation)?
> Danese mentioned that Apache had a process for this. Does anyone know this process?

Not much of a process that I'm aware of. However there are two things speaker-related that indeed did help me at Apache:

a) There's any easy to use Map of ASF people with a link to a speaker search: https://community.zones.apache.org/map.html All data on that site is provided voluntarily, essentially it's a FOAF file you fill in as a committer(?), put it behind some public URL and then tell some csv file within Apache where to find your FOAF file. 
b) There is a media training that Sally Khudairi regularly does during ApacheCons. That's mostly to enable projects to effectively talk to press representatives, write press releases, talk with analysts and give effective talks about their project. Typically it is IIRC a two day training, for free for ASF members.
c) Sally also receives media requests which she then relays to ASF members willing to be interviewed through the general member list.

**Comment by claredillon** [comment]
Agreed in Marketing WG meeting of 31/08 that we will document the process around how to handle speaker requests based on how we engaged at last request.

**Comment by claredillon** [comment]
No progress on this for the last two weeks, as the events, summit prep and PR for sponsorship has taken precedence. Hope to get to it in the next week.

**Comment by claredillon** [comment]
Actively using Speaker Bureau channel and have had volunteers for Community Calls and third party events. Still need to write up the proposed process. Hopefully next week!

**Comment by rrrutledge** [comment]
Thank you for your work, @claredillon ❗  Can you give more detail on this item?  Is it still needed?  Thank you!
